docs/docs/notification-center/iframe-embed.md

```diff
@@ -53,7 +53,100 @@ Optionally the embed init script receives a position object, you can use it to s
 </script>
 ```
 
-### Enabling HMAC Encryption
+## Customizing the theme
+
+The notification center component can be customized by passing a `theme` to the init script.
+
+```html
```

Comment:
I think here we can point to https://docs.novu.co/notification-center/react-components#customizing-the-notification-center-theme

So we have a nice table view of all properties in a single place. And here I would remove the implementation of the light dark properties. Wdyt?

Reply:
Yeah, that's a better idea! For the other question - I answered it above with my point of view :)